fadecandy_driver/src/fadecandy_driver.h

```diff
@@ -0,0 +1,98 @@
+#include <array>
+#include <iostream>
+#include <libusb-1.0/libusb.h>
+#include <vector>
+
+#define LEDS_PER_PACKET 21
+#define LEDS_PER_PACKET 21
+#define LOOKUP_VALUES_PER_PACKET 31
+#define LOOKUP_VALUES_PER_CHANNEL 257
+#define USB_PACKET_SIZE 64
+#define PACKET_TYPE_VIDEO 0x00
+#define PACKET_TYPE_LUT 0x40
+#define FINAL_PACKET_BIT 0x20
+
+#define USB_PRODUCT_ID 0x607a
+#define USB_VENDOR_ID 0x1d50
+#define USB_ENDPOINT 1
+#define INTERFACE_NO 0x01
+
+#define LEDS_PER_STRIP 64
+#define NUM_STRIPS 8
+
+namespace fadecandy_driver
+{
+//!
+//! \brief The FadecandyDriver class
+//!
+class FadecandyDriver
+{
+public:
+  //!
+  //! \brief The color struct contains the r,g and b colors
+  //!
+  struct colors
+  {
+    double r;
+    double g;
+    double b;
+  };
+  libusb_device* fadecandy_device = NULL;
```

Comment:
Doe these all need to be members?

Reply:
Yes all these variables are used in may parts of the program